plugins/importexport/csv/CSVImportExportPlugin.php

```diff
@@ -134,7 +133,12 @@ public function executeCLI($scriptName, &$args)
                     if (!is_array($supportedLocales) || count($supportedLocales) < 1) {
                         $supportedLocales = [$press->getPrimaryLocale()];
                     }
-                    $authorGroup = $userGroupDao->getDefaultByRoleId($press->getId(), Role::ROLE_ID_AUTHOR);
+                    $authorGroup = Repo::userGroup()->getMany(
```

Comment:
This needs to be updated for https://github.com/pkp/pkp-lib/issues/8092.

Reply:
fixed